ci/unit/test_main.sh

```diff
@@ -43,6 +48,6 @@ rsync -rlpt -z -e "ssh $SSH_KEY_ARGS" . jenkins@"$NODE":build/
 ssh -tt "$SSH_KEY_ARGS" jenkins@"$NODE" "HOSTNAME=$HOSTNAME        \
                                          HOSTPWD=$PWD              \
                                          WITH_VALGRIND=$WITH_VALGRIND \
-                                         BULLSEYE=$BULLSEYE        \
+                                         HTTPS_PROXY=\"${HTTPS_PROXY:-}\" \
```

Comment:
> ShellCheck
>
> BULLSEYE appears unused. Verify use (or export if used externally). [SC2034]

Maybe jsut remove all the BULLSEYE references here?

Reply:
If we can, we should.   There may be other issues showing up if the BULLSEYE symbol is missing that ShellCheck does not find.